community/learn/hasura/graphql-tutorial-backend/tutorial-site/content/custom-business-logic.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Custom Business Logic"
+---
+
+Hasura gives you CRUD + realtime GraphQL APIs with authorization & access control. However there are cases where you would want to query something which is not in your database.
+
+Custom business logic can be handled in two ways using Hasura:
+- Writing custom GraphQL resolvers and adding it as a remote schema.
+- Creating event triggers that capture events happening on the specified tables.
```

Comment:
The intent is buried under too much jargon. It should ideally say:

'After a mutation, trigger a webhook'.